Copy edit pass (#73)
ember-learn/super-rentals-tutorial: src/chapters/06-interactive-components.md

ember server
```

In this chapter, you will learn about:
* Adding behavior to components with classes
* Accessing instance variable states from templates
* Managing state with tracked properties
* Using conditionals syntaxes in templates
* Responding to user interaction with actions
* Invoking element modifiers
* Testing user interactions

## Adding Behavior to Components with Classes

So far, all the components we have written are purely *[presentational][TODO: link to presentational]* &mdash; they are simply reusable snippets of markup. That's pretty cool! But in Ember, components can do so much more.

Sometimes, you want to associate some *[behavior][TODO: link to behavior]* with your components so that they can do more interesting things. For example, `<LinkTo>` can respond to clicks by changing the URL and navigating us to a different page.

Here, we are going to do just that! We are going to implement the "View Larger" and "View Smaller" functionality, which will allow our users to click on a property's image to view a larger version, and click on it again to return to the smaller version.

> In other words, we want a way to _toggle_ the image between one of the two *[states][TODO: link to states]*. In order to do that, we need a way for the component to store two possible states, and to be aware of which states it is currently in.

Reviewer: I think this actually want to say "which state it is currently in" (singular "state")

Here, in the *[component's constructor][TODO: link to component's constructor]*, we *[initialized][TODO: link to initialized]* the *[instance variable][TODO: link to instance variable]* `this.isLarge` with the value `false`, since this is the default state that we want for our component.

> ## Accessing Instance Variable States from Templates

Reviewer: "Instance Variable States" is redundant, either "Instance Variables" or "Instance States" are fine (probably the latter).

+{{/if}}
```

> In the template, we have access to the component's instance variables. The `{{#if ...}}...{{else}}...{{/if}}` *[conditionals][TODO: link to conditionals]* syntax allows us to render different content based on a condition (in this case, the value of the instance variable `this.isLarge`). Note that we access instance variables in the template with `this.*`, not the `@argument` syntax you've seen for passed in arguments. Combining these two features, we can render either the small or the large version of the image accordingly.

Reviewer: ```suggestion
In the template, we have access to the component's instance variables, using the same `this.isLarge` syntax. The `{{#if ...}}...{{else}}...{{/if}}` *[conditionals][TODO: link to conditionals]* syntax allows us to render different content based on a condition. In this case, the condition happens to be the `this.isLarge` instance variable. Combining these two features, we can render either the small or the large version of the image accordingly.
```

I am not sure that it is a good idea to call out "as opposed to the `@argument` syntax". It is actually a bit confusing for me to think of them as related at all. I understand where you are coming from, since in old-Ember everything is smooshed together, but I don't know if this is a thought that would come up for a new user reading this tutorial.
